pylint/exceptions.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
Yep, I just added them, do a git pull and it should be everywhere.

Take a look at checkers/spelling.py, I think it should be in the same line.

On Jul 19, 2016 8:48 PM, "Glenn Matthews" <notifications@github.com> wrote:

> In pylint/exceptions.py
> <https://github.com/PyCQA/pylint/pull/1022#discussion_r71388174>:
>
> > @@ -0,0 +1,14 @@
> > +# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
>
> No idea. :-) I only see copyright headers on a few files in the whole repo
> - are you sure this is needed?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/PyCQA/pylint/pull/1022/files/90118b5cc46639b42bf997c343a59293bb42da02#r71388174>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAbWphqQvPoarUaH3iLnqv-z0p32cGHbks5qXQ3zgaJpZM4JOukD>
> .
>